### ISSUE TITLE
Weed out issues in the Candidate List that have already been worked on.

### ISSUE THREAD

**Issue by Elukoye** [publiclab/plots2#9529]
Hi, this is a [first-timers-only issue](https://code.publiclab.org/). This means we've worked to make it more legible to folks who either **haven't contributed to our codebase before, or even folks who haven't contributed to open source before**. 

If that's you, we're interested in helping you take the first step and can answer questions and help you out as you do. Note that we're especially interested in contributions from people from groups underrepresented in free and open-source software!

We know that the process of creating a pull request is the biggest barrier for new contributors. This issue is for you 💝

If you have contributed before, **consider leaving this one for someone new** and looking through our general [help wanted](https://github.com/publiclab/plots2/labels/help-wanted) issues. Thanks!

### 🤔 What you will need to know.

Nothing. This issue is meant to welcome you to Open Source :) We are happy to walk you through the process.

### 📋 Step by Step

- [ ] 🙋 **Claim this issue**: Comment below. If someone else has claimed it, ask if they've opened a pull request already and if they're stuck -- maybe you can help them solve a problem or move it along!

- [ ] The aim of this Issue is to weed out issues that have already been fixed in the Candidate for FTO list on the PublicLab website. Below is how to do it:

- [ ] Go to [the Publiclab website](https://code.publiclab.org/)
- [ ] Scroll to the Candidates section.
- [ ] Go through the list to find issues that have been fixed by reading through the issue thread.
- [ ] If fixed,  write down the issue number here, as a comment
- [ ] 🏁 **Done** Ask in comments for a review :)

Since no code is being changed, we don't need to create a pull request for this issue. We just need to weed out the fixed issues for the FTO candidate list.

[See this page](https://code.publiclab.org/#r=all) for some help in taking your first steps!

### Please keep us updated

💬⏰ - We encourage contributors to be respectful to the community and provide an update **within a week** of claiming the first-timers-only issue. We're happy to keep it assigned to you as long as you need if you update us with a request for more time or help, but if we don't see any activity a week after you claim it we may reassign it to give someone else a chance. Thank you in advance! 

If this happens to you, don't sweat it! Grab another open issue.

### Is someone else already working on this?

🔗- We encourage contributors to link to the original issue in their pull request so all users can easily see if someone's already started on it. 

👥- **If someone seems stuck, offer them some help!** Otherwise, [take a look at some other issues you can help with](https://code.publiclab.org/#r=all). Thanks!

### 🤔❓ Questions?

Leave a comment below!

**Comment by willow19** [comment]
@Elukoye can I try? 

Also I do not see any issues under 'candidates' [here](https://code.publiclab.org/). Should I instead look for the issues [here](https://github.com/publiclab/plots2/labels/first-timers-only)?

**Comment by TildaDares** [comment]
@willow19 Yeah you're right. You should look under the issues section.

**Comment by Elukoye** [comment]
@willow19 ,hi ,yes go ahead 🚀✊. 
Also the candidates list isn't on the publiclab website , [it's here ](https://code.publiclab.org/#r=all).
Scroll down till you see the Candidates heading ,there are issues there that have already been fixed.

**Comment by willow19** [comment]
@RuthNjeri can you please have a look at the list below. I need help with the 'Need Confirmation' list to be sure they are done. I have given brief comments on all open items in the 3rd portion 'Overview on all open items'

### Done
1. [395](https://github.com/publiclab/community-toolbox/issues/395) - **done** in PR [424](https://github.com/publiclab/community-toolbox/pull/424)
2. [71](https://github.com/publiclab/inline-markdown-editor/issues/71) - **done** in [PR 
](https://github.com/publiclab/inline-markdown-editor/pull/88)
3. [4624](https://github.com/publiclab/plots2/issues/4624) - **done** alongwith this [4594](https://github.com/publiclab/plots2/issues/4594) is **done** too
4. [108](https://github.com/publiclab/community-toolbox/issues/108) - **done** in [PR](https://github.com/publiclab/community-toolbox/pull/266)
5. [3865](https://github.com/publiclab/plots2/issues/3865) - **done**
6. [3861](https://github.com/publiclab/plots2/issues/3681) - **done** , there are additional tasks which can be made out of this
7. [3549](https://github.com/publiclab/plots2/issues/3549) - **done** in [PR](https://github.com/publiclab/plots2/pull/7282)

### Need confirmation
1. [69](https://github.com/publiclab/community-toolbox/issues/69) - seems **done**, @jywarren can you please confirm if this can be closed
2. [3513](https://github.com/publiclab/plots2/issues/3513) - not sure if this is done. e.g. PR [9384](https://github.com/publiclab/plots2/pull/9384) has media demo-ing new feature
3. [3460](https://github.com/publiclab/plots2/issues/3460) - seems **done**, no more work seems pending
4. [3458](https://github.com/publiclab/plots2/issues/3458) - seems **done** - looks more like guidelines than a task

### Overview on all open items
1. [9533](https://github.com/publiclab/plots2/issues/9533) has been split up into [9535](https://github.com/publiclab/plots2/issues/9535) and [9539](https://github.com/publiclab/plots2/issues/9539). If both of those are solved, should we mark 9533 as done?
2. [8731](https://github.com/publiclab/plots2/issues/8731) - not apparent that it has been solved
3. [1757](https://github.com/publiclab/image-sequencer/issues/1757) - not solved
4. [7798](https://github.com/publiclab/plots2/issues/7798) - does not seem solved
5. [1569](https://github.com/publiclab/image-sequencer/issues/1569) - a volunteer wants go ahead
6. [315](https://github.com/publiclab/community-toolbox/issues/315) - has a linked break-me-up task [450](https://github.com/publiclab/community-toolbox/issues/450) - if 450 is broken up we'd have converted this candidate into an FTO
7. [310](https://github.com/publiclab/community-toolbox/issues/310) - many volunteers, but can't see the main guy assigned
8. [1355](https://github.com/publiclab/image-sequencer/issues/1355) - not solved
9. [284](https://github.com/publiclab/community-toolbox/issues/284) - has an open [PR](https://github.com/publiclab/community-toolbox/pull/296) but no activity there
10. [6391](https://github.com/publiclab/plots2/issues/6391) - has been broken up into FTOs [6834](https://github.com/publiclab/plots2/issues/6834) (closed) and [8587](https://github.com/publiclab/plots2/issues/8587) (closed) - **done**
11. [574](https://github.com/publiclab/mapknitter/issues/574) - not done
12. [879](https://github.com/publiclab/image-sequencer/issues/879) - some bugs have been reported for this one. Can some of them be converted to FTOs?
13. [178](https://github.com/publiclab/leaflet-blurred-location/issues/178) - not done
14. [135](https://github.com/publiclab/community-toolbox/issues/135) - waiting on @jywarren to review [PR](https://github.com/publiclab/community-toolbox/pull/136)
15. [4178](https://github.com/publiclab/plots2/issues/4178) - open, a volunteer needs go ahead
16. [4092](https://github.com/publiclab/plots2/issues/4092) - not done
17. [3523](https://github.com/publiclab/plots2/issues/3523) - not done

**Comment by willow19** [comment]
Thanks @RuthNjeri , sure I'll pick up another issue. Thanks again for the appreciation!
